src/sonic-yang-models/yang-models/sonic-feature.yang

```diff
@@ -5,11 +5,19 @@ module sonic-feature{
     namespace "http://github.com/Azure/sonic-feature";
     prefix feature;
 
-    import sonic-types {
-        prefix stypes;
+    description "Feature Table yang Module for SONiC";
+    
+    typedef feature-state {
+        type string {
+            pattern "enabled|disabled|always_enabled|always_disabled";
+        }
```

Comment:
Please add short description of the states as well.

Reply:
added in latest commit